views/overview/list-categories.php

```diff
@@ -44,10 +45,16 @@
                         $module = Yii::$app->getModule('wiki');    
                         $user = Yii::$app->user->identity;
                         $numberingEnabled = $module->settings->contentContainer($user)->get('overviewNumberingEnabled');
+                        $editingEnabled = $module->settings->contentContainer($user)->get('wikiTreeEditingEnabled', FALSE);
                     ?>
                     <a href="<?= Url::current(['toggle-numbering']) ?>" class="btn btn-info btn-sm toggle-numbering">
                         <?= $numberingEnabled ? Yii::t('WikiModule.base', 'Disable Numbering') : Yii::t('WikiModule.base', 'Enable Numbering') ?>
                     </a>
+                    <?php if ($contentContainer->can(AdministerPages::class)): ?>
+                        <a href="<?= Url::current(['toggle-wiki-tree-editing']) ?>" class="btn btn-info btn-sm toggle-editing">
+                            <?= $editingEnabled ? Yii::t('WikiModule.base', 'Disable wiki tree editing') : Yii::t('WikiModule.base', 'Enable wiki tree editing') ?>
+                        </a>
+                    <?php endif; ?>
```

Comment:
The toggle buttons already take up a lot of space. Therefore, please nest them in a dropdown. You can call them options, for example.